.github/workflows/ci.yml

```
@@ -3,8 +3,38 @@ name: CI/CD Pipeline
+      - 'infra/**'
+      - 'scripts/**'
+      - 'docker/**'
+      - 'docker-compose*.yml'
+      - 'pyproject.toml'
+      - 'uv.lock'
+      - 'go.mod'
+      - 'go.sum'
+      - 'package.json'
+      - 'bun.lock'
   pull_request:
     branches: [ main, develop ]
```

YAMLlint (1.38.0)

[error] too many spaces inside brackets

(brackets)